dev/reference/tools/loader.md

```diff
@@ -97,6 +97,14 @@ user = "root"
 password = ""
 port = 4000
 
+# sql_mode used when loading data. If the `sql-mode` value is not provided or set to "@DownstreamDefault", downstream global sql_mode will be used
```

Comment:
Are `sql_mode` and `sql-mode` used interchangeably? If not, please frame `sql_mode` with back quotes.

Reply:
`sql_mode` is used in TiDB, `sql-mode` is used in  loader config. Both of them are added back quotes wrapping